go.sum

```diff
@@ -1,6 +1,5 @@
 github.com/Masterminds/semver v1.5.0 h1:H65muMkzWKEuNDnfl9d70GUjFniHKHRbFPGBuZ3QEww=
```

Comment:
I'm surprised at the number of transitive dependencies autogold is adding. The use case you have here is literally just a few lines of go.

Reply:
Yeah, I was surprised, too. I think the use case for autogold is perfect though. It is really snapshotting the UI.